### PR TITLE
pr132

### PR DESCRIPTION
**Pull request by zzzeek**
change some things.

### CODE REVIEW

**Comment by sqla-tester** [comment]
New Gerrit review created for change a0e3686b7471fec3cfc4c85b48655bad26b7cec9: https://gerrit.sqlalchemy.org/#/c/sqlalchemy/testgerrit/+/1045

File: foo.py [hunk 1]
```diff
@@ -3,14 +3,14 @@
 from foo import bar
 from foo import bat
 
-import io  # noqa
```
Comment (sqla-tester) [inline foo.py]: **mike bayer** (zzzeek) wrote:

review import removal

File: foo.py [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
 import re  # noqa
 
 
-def foo_one(x, y):
+def foo_one(x, y, z):
     """do foo one"""
     bar.do_x(x)
     bar.do_y(y)
+    bat.do_z(z)
```
Comment (sqla-tester) [inline foo.py]: **mike bayer** (zzzeek) wrote:

review line z

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

Patch Set 1:

(1 comment)

File: foo.py (hunk 2 above)
Comment (sqla-tester) [reply foo.py]: **mike bayer** (zzzeek) wrote:

Done

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

Patch Set 1:

so can you do this

**Comment by sqla-tester** [comment]
**"mike bayer <>"** (zzzeek) wrote:

test abandon

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/1045 has been **abandoned**.  That means that at least for the moment I need to close this pull request. Sorry it didn't work out :(

**Comment by sqla-tester** [comment]
**"mike bayer "** (zzzeek) wrote:

test restore

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/1045 has been **restored**.  That means I can reopen this pull request!  Hooray :)

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

test abandon again

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/1045 has been **abandoned**.  That means that at least for the moment I need to close this pull request. Sorry it didn't work out :(

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

latest and greatest comment.

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

restored!

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/1045 has been **restored**.  That means I can reopen this pull request!  Hooray :)

File: bar.py [hunk 3]
```diff
@@ -13,14 +13,14 @@ def bar_one(x, y):
...
-        bat.do_q(q)
-    else:
-        bat.do_q(q + 7)
+    bat.do_q(q + name)
 
 
 def bar_three(a, g, h):
     bat.do_a(a)
     bat.do_g(g)
-    bat.do_h(h)
```
[the remark this answers is not in the document]
Comment (zzzeek) [reply bar.py]: github reply number one

[the remark this answers is not in the document]
Comment (zzzeek) [reply bar.py]: turned on the API comments, and... github reply number two

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

(1 comment)

File: bar.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (sqla-tester) [reply bar.py]: **mike bayer** (zzzeek) wrote:

**any markup** ?   [links](http://sqlalchemy.org) ?

hmm

[the remark this answers is not in the document]
Comment (zzzeek) [reply bar.py]: well this is cool

File: foo.py [hunk 4]
```diff
@@ -3,14 +3,14 @@
 from foo import bar
 from foo import bat
 
-import io  # noqa
 import re  # noqa
 
 
-def foo_one(x, y):
+def foo_one(x, y, z):
```
Comment (zzzeek) [inline foo.py]: github review comment one

File: foo.py [hunk 5]
```diff
@@ -23,6 +23,7 @@ def foo_two(z, x, y):
 def foo_three(a, g, j):
     """do foo three"""
     bat.do_a(a)
+    bat.do_g(g)
```
Comment (zzzeek) [inline foo.py]: gituhb review comment two

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

> (1 comment)
 > 
 > zzzeek@github wrote:
 > 
 > (1 comment(s) from GitHub)

oh yeah you think

File: foo.py [hunk 6]
```diff
@@ -3,14 +3,14 @@
...
 
-import io  # noqa
 import re  # noqa
 
 
-def foo_one(x, y):
+def foo_one(x, y, z):
     """do foo one"""
     bar.do_x(x)
     bar.do_y(y)
```
Comment (sqla-tester) [inline foo.py]: **mike bayer** (zzzeek) wrote:

sure

File: foo.py (hunk 2 above)
Comment (sqla-tester) [reply foo.py]: **mike bayer** (zzzeek) wrote:

yup

File: foo.py (hunk 5 above)
Comment (zzzeek) [reply foo.py]: one more test

Comment (zzzeek) [reply foo.py]: are we online

Comment (zzzeek) [reply foo.py]: are we online again

Comment (sqla-tester) [reply foo.py]: **mike bayer** (zzzeek) wrote:

yes we are online

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

(1 comment)

File: foo.py (hunk 5 above)
Comment (zzzeek) [reply foo.py]: ok no that wasn't quite online

Comment (sqla-tester) [reply foo.py]: **mike bayer** (zzzeek) wrote:

NOW we are online

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

(1 comment)

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

ok try another one

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/1045 has been **abandoned**.  That means that at least for the moment I need to close this pull request. Sorry it didn't work out :(

**Pull request closed without merging**